bundles/org.openhab.binding.mqtt.homeassistant/src/main/java/org/openhab/binding/mqtt/homeassistant/internal/handler/HomeAssistantThingHandler.java

```diff
@@ -120,7 +124,7 @@ public HomeAssistantThingHandler(Thing thing, MqttChannelTypeProvider channelTyp
                 this.transformationServiceProvider);
     }
 
-    @SuppressWarnings({ "null", "unused" })
+    @SuppressWarnings({ "null" })
```

Comment:
And this one, you keep it for what reason ?

Reply:
To be honest, I adressed only the warnings, not existing supress statements.

The "unused" bit raised warning in eclipse...

Reply:
As it is not something you added in that PR, let's say it is OK for me.